src/css/structure/modules/infragistics.ui.tree.css

```diff
@@ -125,6 +125,21 @@
 	height:1.6em;
 }
 
+/* Marin Popov, 26.06.2017, Bug 236973 - Checkboxes cannot be clicked */
+.ui-igtree  .ui-igcheckbox-normal-on {
+	position:relative;
+	top:0;
+	left:0;
+	margin-top:0;
+	margin-left:0;
+}
+.ui-igtree .ui-igcheckbox-normal-off,
+.ui-igtree .ui-igcheckbox-normal-on {
+	width:100%!important;
```

Comment:
@desig9stein Can you experiment with removing the !important in this general definition. Let's see if that would break the font icons, and if not then we should get rid of it :)